### ISSUE TITLE
could not load AlphaVSS

### ISSUE THREAD

**Issue by Zelss** [supermemo/SuperMemoBackup#2]
![Screen Shot 2020-03-07 at 7 08 01 PM](https://user-images.githubusercontent.com/47744078/76144954-f9ccaf00-60a6-11ea-8633-975b90cc322e.png)

**Comment by alexis-** [comment]
VC Redists ?

**Comment by Zelss** [comment]
Installing Visual C++ x64 solved the problem.

**Comment by rajlego** [comment]
Had same issue and same solution for parallels. Would be good to add to the wiki.

**Comment by Zelss** [comment]
Yes bitshelter only works on 64bit windows as well not on 32 bit.
